modules/virt-installing-qemu-guest-agent-on-linux-vm.adoc

```diff
@@ -7,10 +7,9 @@
 [id="virt-installing-qemu-guest-agent-on-linux-vm_{context}"]
 = Installing QEMU guest agent on a Linux virtual machine
 
-The `qemu-guest-agent` is widely available and available by default in Red Hat
-virtual machines. Install the agent and start the service.
+The `qemu-guest-agent` is widely available and available by default in {op-system-base-full} virtual machines (VMs). Install the agent and start the service.
 
-To check if your virtual machine (VM) has the QEMU guest agent installed and running, verify that `AgentConnected` is listed in the VM spec.
+To check if your VM has the QEMU guest agent installed and running, verify that `AgentConnected` is listed in the VM spec.
```

Comment:
This could be moved to the end of the installation procedure, as a verification.